staking/keeper_test.go

```diff
@@ -308,7 +308,7 @@ func (suite *KeeperTestSuite) TestDividendAccountTree() {
 	divAccounts := make([]hmTypes.DividendAccount, 5)
```

Comment:
Please move `dividend-account` related tests to `topup`. 
staking module doesn't hold `dividend-accounts` anymore